core/settings.py

```diff
@@ -69,7 +69,7 @@ class Common(Configuration):
 
     WSGI_APPLICATION = "core.wsgi.application"
 
-    default_db = "sqlite:///" + os.path.join(BASE_DIR, "db.sqlite3")
+    default_db = "postgres://USER:PASSWORD@HOST:PORT/NAME"
```

Comment:
Acho que seria legal a gente deixar um valor padrão mesmo, pra quando a pessoa rodar já funcionar (meio na linha do que o @cuducos sugeriu também). Algo assim:

```suggestion
    default_db = "postgres://postgres:postgres@localhost:5432/mariaquiteria"
```

Seria o mesmo para o `.env.example`. Aí no README a gente pode explicitar que esse é o valor padrão e que pode ser modificado via variável de ambiente. O que acham?

Reply:
```suggestion
    default_db = os.environ.get("DATABASE_URL")
```

Não seria melhor ler a variábel de ambiente?

Reply:
Na real, acho que ele já pega essa variável de ambiente por padrão e se não acha usa esse `default` (ACHO pq agora não dá pra eu testar):

https://github.com/jacobian/dj-database-url/blob/1937ed9e61d273163353c8546825dd529ce8546c/dj_database_url.py#L59-L64

Reply:
Devo ir pela sugestão da @anapaulagomes então? Faz sentido o que ela comentou.

Reply:
Testei aqui da seguinte forma:
* Sem a variável de ambiente configurada
* Com a linha: `default_db = "postgres://postgres:postgres@localhost:5432/mariaquiteria"`

```
In [6]: import os                                                                                                                                                            

In [7]: os.getenv("DATABASE_URL") is None                                                                                                                                    
Out[7]: True

In [8]: from django.conf import settings                                                                                                                                     

In [9]: settings.DATABASES                                                                                                                                                   
Out[9]: 
{'default': {'NAME': 'mariaquiteria',
  'USER': 'postgres',
  'PASSWORD': 'postgres',
  'HOST': 'localhost',
  'PORT': 5432,
  'CONN_MAX_AGE': 0,
  'ENGINE': 'django.db.backends.postgresql_psycopg2',
  'ATOMIC_REQUESTS': False,
  'AUTOCOMMIT': True,
  'OPTIONS': {},
  'TIME_ZONE': None,
  'TEST': {'CHARSET': None, 'COLLATION': None, 'NAME': None, 'MIRROR': None}}}
```

Seguro dizer que dessa forma funciona. :) (Ana, perdão pela demora)